src/demos/Banners.vue

```diff
@@ -446,7 +441,7 @@ export default class Banners extends Vue {
   }
 
   test() {
-    console.log("test");
+    // console.log("test");
```

Comment:
consider removing

Reply:
Going to keep for now until I fully finish adding accessibility to all components.